docs/guides/isolated_systems.md

```diff
@@ -0,0 +1,203 @@
+
+The classification of changes as breaking/non-breaking in the non-production system plan (step 2) will not be available to the production system plan (step 3) because the systems do not share SQLMesh state data. Therefore, the classification must occur again in the production system.
+
+In isolated systems, SQLMesh's virtual data environments will operate normally *within* each system, but not across systems. For example, when the non-production system's staging environment is created (step 2c), any computations previously run in a developer's personal environment will not run again.
+
+In step (3) above, the creation of the `staging` environment in the production system will require running all the computations required by code changes. Deployment to the production system's `prod` environment (step 4) will not require additional computation due to SQLMesh's virtual data environments. The deployment only requires updates views so will be very rapid.
+
+## SQLMesh features
+
+This section describes how two key SQLMesh features perform when a project runs on isolated systems: virtual data environments and computing only what's necessary.
+
+### Virtual data environments
+
+SQLMesh's virtual data environments ensure that computations are only run when necessary, saving analyst time and reducing computing expenses.
+
+Environments are isolated namespaces where users can safely modify code and examine the outputted data. The production environment runs the models your business relies on for its operations, and other environments are created by users on demand.
+
+SQLMesh environments work by separating database objects into two conceptual "layers": physical and virtual. The physical layer consists of the tables created, modified, and queried by SQLMesh models. The virtual layer consists of [views](https://en.wikipedia.org/wiki/View_(SQL)) that pass through data from an underlying physical table via `SELECT *`.
+
+When an existing model is modified, a new physical table is created to reflect the updated model. The virtual layer of any environment containing that version of the model can point to the same new physical table. This approach allows a model version's computations to run once and be reused by any environment (including production).
+
+#### Table reuse in isolated systems
+
+Isolated production systems cannot reuse the physical tables created in non-production systems. Therefore, computations performed in non-production systems must be run again when an updated model version is pushed to the production system.
+
+However, computations performed in different SQLMesh environments inside a system will be reused by other SQLMesh environments in that system. In organizations with many collaborators, most computations occur during development, resulting in time and cost savings due to table reuse within the non-production system.
+
+### Compute only what's necessary
```

Comment:
These last two sections felt a little redundant, but I may be biased. After reading the whole thing, it seemed to me that it'd already been explained how SQLMesh only computes what's necessary, eg. L117, L143, L173, L195.